example/src/components/ArrowBoxExamples.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import ArrowBox from '../../../src/components/ArrowBox';
+import Code from '../../../src/components/Code';
+import Colors from '../../../src/components/Colors';
+
+export default function ArrowBoxExamples() {
+  return (
+    <section className="_full-width-row" id="ArrowBox">
+      <div className="_container _container_large">
+        <h2 className="base--h2">Arrow Box</h2>
+        <div className="row">
+          <div
+            style={{
+              position: 'relative',
+              width: '10rem',
+              margin: 'auto',
+              backgroundColor: Colors.gray_10,
+              display: 'block',
+              padding: '1rem',
+            }}
+          >
+            This is the element that the ArrowBox is anchored to.
+            <ArrowBox
+              direction="top"
+              show
```

Comment:
I would say make this `show={true}` just to make it more clear. Otherwise, someone might believe that they need to remove the attribute to hide the ArrowBox.